README.md

```diff
@@ -6,7 +6,9 @@ This library provides a [DynamoDB](https://aws.amazon.com/dynamodb/)-backed pers
 
 This version of the library requires at least version 5.3.0 of the LaunchDarkly Go SDK. In Go SDK versions prior to 5.0.0, the `lddynamodb` package was built into the SDK (`gopkg.in/launchdarkly/go-server-sdk.v4/lddynamodb`).
 
-The minimum Go version is 1.14.
+This version of the library uses the [v2 AWS Go SDK](https://github.com/aws/aws-sdk-go-v2).
+
+The minimum Go version is 1.16.
```

Comment:
1.16 is the lowest version supported by the current Go SDK and ld-relay. We can bump it to 1.18 in our next major version of this library which will be for the next major version of the Go SDK.